ext/Recorder.php

```diff
@@ -16,8 +16,6 @@
  * Saves a screenshot of each step in acceptance tests and shows them as a slideshow on one HTML page.
  * Activated only for suites with WebDriver module enabled.
  *
- *  ![recorder](http://codeception.com/images/recorder.gif)
```

Comment:
Then please make it as a link.
Recorder is much easier to understand when you see it in action.